welcome/src/main/kotlin/edu/artic/welcome/WelcomeFragment.kt

```diff
@@ -35,5 +50,61 @@ class WelcomeFragment : BaseViewModelFragment<WelcomeViewModel>() {
             }
         }
 
+        val list = getTours()
+
+        context?.let {
+            tourSummaryRecyclerView.layoutManager = LinearLayoutManager(activity, LinearLayoutManager.HORIZONTAL, false)
+            tourSummaryRecyclerView.adapter = ToursAdapter(list, it)
+            welcomePreferencesManager = WelcomePreferencesManager(it)
+        }
+
+        /**
+         * Peak Animation.
+         * Scroll RecyclerView to the last item and back again to first item.
+         */
+        val peekedTourSummary = welcomePreferencesManager?.peekedTourSummary ?: false
+
+        if (!peekedTourSummary) {
+            handler.postDelayed({
+                tourSummaryRecyclerView.post {
+                    tourSummaryRecyclerView.smoothScrollToPosition(list.count() - 1)
+                }
+            }, 2000)
+
+            handler.postDelayed({
+                tourSummaryRecyclerView.post {
+                    tourSummaryRecyclerView.smoothScrollToPosition(0)
+                    welcomePreferencesManager?.peekedTourSummary = true
+                }
+            }, 3000)
+        }
+    }
+
+
+    private fun getTours(): List<ArticTour> {
+
+        return activity.let {
+            if (it == null) {
+                emptyList<ArticTour>()
+            } else {
+                val toursJson = it.assets.fileAsString("json", "tours.json")
+                val adapter: JsonAdapter<List<ArticTour>> = moshi.adapter(Types.newParameterizedType(List::class.java, ArticTour::class.java))
+                return@let adapter.fromJson(toursJson) as List<ArticTour>
+            }
+
+        }
+    }
+
+    override fun onDestroy() {
+        super.onDestroy()
+        handler.removeCallbacksAndMessages(null)
+    }
+
+}
+
+fun AssetManager.fileAsString(subdirectory: String, filename: String): String {
```

Comment:
I know its temporary, but maybe still belongs in a utility file in `base`?

Reply:
Moved.